modules/tests/purser-trezor/staticMethods/signTransaction.test.js

```
@@ -189,6 +193,9 @@ describe('`Trezor` Hardware Wallet Module Static Methods', () => {
       );
     });
     test('Signs a transaction without a destination address', async () => {
+      payloadListener.mockImplementation(() => ({
```

This test was silently failing :/

---

Nice catch!